.chloggen/dockerapiversion.yaml

```diff
@@ -0,0 +1,27 @@
+# Use this changelog template to create an entry for release notes.
+
+# One of 'breaking', 'deprecation', 'new_component', 'enhancement', 'bug_fix'
+change_type: bug_fix
```

Comment:
To clarify. this isn't a breaking change user change since the existing behavior persists but is expanded to allow the workaround. It is a breaking api change though if there are direct consumers of the receiver or extension, but I'm not sure signaling as such in the changelog is desirable.

Reply:
agreed, since you have `change_logs: [user, api]` as it should be, it's considered a breaking change for api. There's no way to disambiguate between the 2 so I guess we are stuck declaring it `breaking`.

Reply:
You _could_ add two changelog files, one `bug_fix` for `user`, and one `breaking` for `api`. I know this was done [here](https://github.com/open-telemetry/opentelemetry-collector-contrib/pull/29862/), probably more as well.

Reply:
You _could_ add two changelog files, one `bug_fix` for `user`, and one `breaking` for `api`. I know this was done [here](https://github.com/open-telemetry/opentelemetry-collector-contrib/pull/29862/), probably more as well.

Reply:
I'm ok w/ it being a single item, "the best kind of correct" and all.